google/cloud/bigquery/magics/magics.py

```diff
@@ -671,7 +671,10 @@ def _cell_magic(line, query):
                 _handle_error(ex, args.destination_var)
                 return
 
-            result = rows.to_dataframe(bqstorage_client=bqstorage_client)
+            result = rows.to_dataframe(
+                bqstorage_client=bqstorage_client,
+                create_bqstorage_client=use_bqstorage_api,
```

Comment:
Since we already construct a `bqstorage_client` (I believe so we can use a customer user-agent), can't we just always set `create_bqstorage_client` to `False`?

Reply:
Hm... if we don't want to use the BQ Storage API, we set it to `False`, and if we do, we pass in a BQ Storage client instance ourselves, meaning that the value is ignored anyway.

Yeah, always passing `False` is more simple.

Reply:
FWIW, I left the `mock.ANY` assertion in some of the tests, since that's still technically correct (the value is ignored if a BQ Storage client instance is explicitly provided).